### ISSUE TITLE
SCSS files cannot be compiled upon save [nightly build 2.1.7]

### ISSUE THREAD

**Issue by nj** [madskristensen/WebEssentials2013#1187]
When saving a `.scss` file in 2.1.7, it fails compiling the `.css` version with the following stacktrace:

```
18-06-2014 23:15:46: SCSS: unparsable compilation error: module.js:340
    throw err;
          ^
Error: Cannot find module 'node-sass'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (C:\Users\Nikolaj\AppData\Local\Microsoft\VisualStudio\12.0\Extensions\cxcbj4ld.fao\Resources\nodejs\tools\node_modules\node-sass-middleware\middleware.js:1:74)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
18-06-2014 23:15:46: SCSS: admin.scss compilation failed: module.js:340
    throw err;
          ^
Error: Cannot find module 'node-sass'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (C:\Users\Nikolaj\AppData\Local\Microsoft\VisualStudio\12.0\Extensions\cxcbj4ld.fao\Resources\nodejs\tools\node_modules\node-sass-middleware\middleware.js:1:74)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

The issue seems to be related to the installer of the extension ressources which lacks to include the following file:
`Resources\nodejs\tools\node_modules\node-sass\index.js`

As with the other modules, this file normally contains a single line, which in this case would be including something like:
`module.exports = require("./sass.js");`

In other words, the solution is simply to add this file to point to the sass.js file, and the problem is solved - making the builds work again.

Have tried reinstalling 2.1.7 without luck, so it does seems like it for some reason is just missing in this particular build.
